lib/cbus_elixir_web/views/page_view.ex

```diff
@@ -15,7 +19,32 @@ defmodule CbusElixirWeb.PageView do
     "https://www.gravatar.com/avatar/#{hash}"
   end
 
+  def get_next_meeting() do
```

Comment:
This is nitpicky; but `get_` is a smell too :)

Reply:
Fixed